### ISSUE TITLE
It does not work with Ubuntu 20.04 LTS

### ISSUE THREAD

**Issue by josego85** [santiacq/krunnercrypto#5]
Hello:
It does not work with Ubuntu 20.04 LTS.
 The error is: 

```
CMake Error at CMakeLists.txt:8 (find_package):
  Could not find a configuration file for package "ECM" that is compatible
  with requested version "5.72.0".

  The following configuration files were considered but not accepted:

    /usr/share/ECM/cmake/ECMConfig.cmake, version: 5.68.0
```

Thank you

**Comment by santiacq** [comment]
Hello, sorry for the delay in answering.
Currently Ubuntu 20.04 is unsupported, I've tried installing it, and apparently one of the KDE libraries included in 20.04 is older than the one I'm using in the plugin.
I've tested it in Ubuntu 21 and it works correctly.
When I get back from vacation I'll add that it doesn't work in 20.04 to the Readme and see if it's possible to make it work using an older version of ECM.

**Comment by santiacq** [comment]
I tried changing the KDE Framework to version 5.68 in order to make it work but that version doesn't support `K_EXPORT_PLASMA_RUNNER_WITH_JSON`. In conclusion I couldn't make it work :disappointed:. I added to the Readme that Ubuntu 20.10 or greater is required
